### PR TITLE
Create new registrations section

### PR DESCRIPTION
**Pull request by GavT**

### CODE REVIEW

File: registrations.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+## Registrations
```
Comment (tudor-malene) [inline registrations.md]: what about: "Joining the Obscuro Network"

File: registrations.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+For nodes and users to interact with the Obscuro network they must register via the Network Management contract.
```
Comment (tudor-malene) [inline registrations.md]: Anyone wishing to join the Obsuro network as a Node must first check their hardware is compatible with the latest Attestation Constraints. Then they must download and install the latest software, and then they must perform the steps detailed below.

File: registrations.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+* Once in possession of the secret, the TEE can start processing all the L2 transactions that are stored on the L1 blockchain and build the state.
+* Once this is completed, the new aggregator can join the gossip with the other aggregators and participate in the lottery for producing rollups.
+* Some end users will send encrypted instructions directly to this server, and it will gossip these with other nodes in the L2 network, encrypted with the shared secret.
+* As aggregators process messages, they maintain the L2 state in the encrypted TEE memory or encrypted in a local database. If they are the winner of the lottery, when the time comes they will create a valid rollup and publish it to L1.
+* All aggregators keep track of the blocks submitted to the management contract to make sure they are up-to-date with the source of truth.
+* The first aggregator to register has a special role, as it has to create the shared secret.
+
+Note: Each aggregator needs an ETH balance on the L1 to pay for the submission of the rollup.
+
+### User Registration
```
Comment (tudor-malene) [inline registrations.md:46]: I would put this in a separate chapter, now that I look at it.

**Review by tudor-malene — CHANGES_REQUESTED** [review]

File: registrations.md (hunk 3 above)
Comment (GavT) [reply registrations.md:46]: Here we mention "the Obscuro wallet". Should the text refer to an "Obscuro-enabled wallet"? We use this elsewhere in the WP.

Comment (tudor-malene) [reply registrations.md:46]: good point. let's stick with "Obscuro-enabled wallet"

**Pull request merged**